src/v/cloud_topics/reconciler/reconciler.h

```diff
@@ -153,6 +155,7 @@ class reconciler {
     data_plane_api* _data_plane;
     l1::io* _l1_io;
     cluster::metadata_cache* _metadata_cache;
+    l1::metastore* _metastore;
```

Comment:
i don't think this first commit will compile since this would be an unused private member?